db/migrate/20200425033724_create_existing_items.rb

```diff
@@ -0,0 +1,8 @@
+class CreateExistingItems < ActiveRecord::Migration[6.0]
```

Comment:
Should this be named "ExistingTransaction"?

Reply:
I'd generalize it and call it `Event`

Reply:
because the question you're asking is: "Has this event happened already?"

Reply:
Or maybe `ExistingEvent`

Reply:
whatever you think — I think of Transactions as something that you make with the database, and not all calls to our API will lead to a Transaction